packages/grafana-data/src/dataframe/DataFrameJSON.ts

```diff
@@ -20,14 +20,16 @@ export interface DataFrameJSON {
   data?: DataFrameData;
 }
 
+type FieldValues = any[];
```

Comment:
```suggestion
// eslint-ignore-next-line
type FieldValues = any[];
```

Reply:
Or let's use `unknown`. It will force users to deal with it, but then they can cast it to something meaningful.